.md/.project/security.project.md

```
@@ -0,0 +1,37 @@
+
+Supported Versions
+Since DevEnviro is under active development, only the latest version is supported. Security updates will be applied to the most recent commit on the main branch.
+
+Version	Supported
+latest	:white_check_mark:
+
+Export to Sheets
+Reporting a Vulnerability 🛡️
+Please do not report security vulnerabilities through public GitHub issues.
+
+Instead, please send an email to **security@devenviro.example.com**  <!-- update to real mailbox -->
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)